packages/odyssey-react-mui/src/labs/PageTemplate.tsx

```diff
@@ -0,0 +1,249 @@
+   */
+  documentationText?: string;
+  /**
+   * An optional `Drawer` object. Can be of variant 'temporary' or 'persistent'.
+   */
+  drawer?: ReactElement;
+  /**
+   * An optional `Button` object to be situated in the layout header. Should almost always be of variant `primary`.
+   */
+  primaryCallToActionComponent?: ReactElement;
+  /**
+   * An optional `Button` object to be situated in the layout header, alongside the `callToActionPrimaryComponent`.
+   */
+  secondaryCallToActionComponent?: ReactElement;
+  /**
+   * An optional `Button` object to be situated in the layout header, alongside the other two `callToAction` components.
+   */
+  tertiaryCallToActionComponent?: ReactElement;
+  /**
+   * The content of the layout. May be a `string` or any other `ReactNode` or array of `ReactNode`s. Will often be `Grid` objects.
+   */
+  children?: ReactNode;
+  /**
+   * When set to `true`, the layout expands past its max width of 1440px and spans the entire available screen width.
+   */
+  isFullWidth?: boolean;
+};
+
+type TemplateContentProps = {
+  odysseyDesignTokens: DesignTokens;
+  isDrawerOpen?: boolean;
+  drawerVariant?: string;
+};
+
+const TemplateContainer = styled("div", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" && prop !== "isFullWidth",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isFullWidth: boolean;
+}>(({ odysseyDesignTokens, isFullWidth }) => ({
+  maxWidth: isFullWidth
+    ? "100%"
+    : `calc(1440px + ${odysseyDesignTokens.Spacing6} + ${odysseyDesignTokens.Spacing6})`,
+  marginInline: isFullWidth ? odysseyDesignTokens.Spacing6 : "auto",
+  padding: odysseyDesignTokens.Spacing6,
+}));
+
+const TemplateHeader = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>(({ odysseyDesignTokens }) => ({
+  display: "flex",
+  alignItems: "flex-end",
+  justifyContent: "space-between",
+
+  ["@media (max-width: 800px)"]: {
+    alignItems: "flex-start",
+    flexDirection: "column",
+    gap: odysseyDesignTokens.Spacing2,
+  },
+}));
+
+const TemplateHeaderPrimaryContent = styled("div")(() => ({
+  [".MuiTypography-root:last-child"]: {
+    marginBlockEnd: "0",
+  },
+}));
+
+const TemplateHeaderSecondaryContent = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>(({ odysseyDesignTokens }) => ({
+  alignItems: "flex-end",
+  display: "flex",
+  flexDirection: "column",
+  gap: odysseyDesignTokens.Spacing2,
+  minHeight: odysseyDesignTokens.Spacing7,
+  justifyContent: "center",
+
+  ["@media (max-width: 800px)"]: {
+    alignItems: "flex-start",
+  },
+}));
+
+const TemplateHeaderButtons = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>(({ odysseyDesignTokens }) => ({
+  display: "flex",
+
+  ["& > div, & > button"]: {
```

Comment:
@jordankoschei-okta This is already display flex, why not just use gap on the parent, rather than messing with margins here?